jcl/src/java.base/share/classes/com/ibm/jit/JITHelpers.java

```diff
@@ -1031,37 +1043,28 @@ public int getInitialLockword64(long j9clazz) {
 	 * @parm flags
 	 *          Class flags from the J9Class.
 	 *
-	 * @parm lockReservationHistory
-	 *          lockReservationHistory from the J9Class. Contains value of the Reserved Counter and the Cancel Counter.
+	 * @parm reservedCounter
+	 *          reservedCounter from the J9Class.
+	 *
+	 * @parm cancelCounter
+	 *          cancelCounter from the J9Class.
 	 *
 	 * @return
 	 *          The initial lockword to use.
 	 */
-	public int getInitialLockword(int flags, long lockReservationHistory) {
+	public int getInitialLockword(int flags, long reservedCounter, long cancelCounter) {
```

Comment:
Why are the counter parameters of type `long`? They're at most 16 bits - `int` would seem a better choice.

Reply:
I'll change that. I was using `long` there before because `lockReservationHistory` was `long` sometimes but that shouldn't be needed anymore after splitting it up.